### ISSUE TITLE
Fix GQL queries on block fields not being prepped properly

### ISSUE THREAD

**Issue by engram-design** [verbb/vizy#9]
For example, asset fields in blocks return only their ID, not an asset element query.
![image](https://user-images.githubusercontent.com/1221575/117091280-1a24f600-ad9e-11eb-8049-4c49e9a40195.png)

**Comment by huelabs** [comment]
I know you'll probably review all fields but I'm getting back IDs on image, tag, and entries fields.

**Comment by engram-design** [comment]
Yep, working on revamped GQL querying, which is taking a little longer to implement.

**Comment by engram-design** [comment]
Added much better support for GQL for the next release. I would recommend switching to use `nodes` for this, and a few more benefits, rather than using the `rawNodes`, which is only going to return the data stored by Vizy in the database.

To get this early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.2",
  "...": "..."
}
```

Then run `composer update`.

Also have a look at the updated [docs](https://verbb.io/craft-plugins/vizy/docs/developers/graphql)

**Comment by engram-design** [comment]
Fixed in [1.0.3](https://github.com/verbb/vizy/releases/tag/1.0.3)
